src/components/Onboarding/OnboardingScreen.js

```diff
@@ -0,0 +1,668 @@
+    },
+    marginTop: '10%',
+  },
+  greenButton: {
+    backgroundColor: '#53AA48',
+    '&:hover': {
+      backgroundColor: '#53AA48',
+    },
+    marginTop: '10%',
+  },
+  submitButton: {
+    backgroundColor: '#53AA48',
+    '&:hover': {
+      backgroundColor: '#53AA48',
+    },
+    width: '50%',
+  },
+  textBox: {
+    width: '70%',
+  },
+  card: {
+    margin: 16,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  stepper: {
+    margin: 'auto',
+    width: '60%',
+  },
+  icon: {
+    color: 'red !important',
+  },
+});
+const ROLE_TITLES = ['buyer', 'nonprofit', 'seller'];
+const BUTTONS = [
+  {
+    id: 0,
+    title: 'Buyer',
+    image: Fruit3,
+    styling: useStyles.fruit3,
+    fruit: {
+      marginTop: '10%',
+
+      height: '134px',
+      width: '98px',
+      left: '89px',
+      top: '46px',
+    },
+  },
+  {
+    id: 1,
+    title: 'Non-profit or Agency',
+    image: Fruit2,
+    styling: useStyles.fruit2,
+
+    fruit: {
+      marginTop: '10%',
+      height: '130px',
+      width: '105px',
+      left: '96px',
+      top: '50px',
+    },
+  },
+  {
+    id: 2,
+    title: 'Seller',
+    image: Fruit1,
+    styling: useStyles.fruit1,
+
+    fruit: {
+      marginTop: '10%',
+      height: '141px',
+      width: '91px',
+      left: '72px',
+      top: '39px',
+      // border-radius: '0px',
+    },
+  },
+];
+
+const INITIAL_FORM_STATE = {
+  firstName: '',
+  lastName: '',
+  email: '',
+  password: '',
+  confirmPassword: '',
+  org: '',
+  phone: '',
+  zipcode: '',
+  comments: '',
+};
+
+function getSteps() {
+  return ['Step 1', 'Step 2', 'Step 3', 'Confirmation'];
+}
+
+export default function OnboardingScreen() {
+  const [formState, setFormState] = useState(INITIAL_FORM_STATE);
+  const [role, setRole] = useState(-1);
+  const [loading, setLoading] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const classes = useStyles();
+
+  const handleChange = useCallback((event) => {
+    event.preventDefault();
+    setFormState(
+      {
+        ...formState,
+        [event.currentTarget.name]: event.currentTarget.value,
+      },
+    );
+  }, [formState, setFormState]);
+
+  const handleSubmit = useCallback((event) => {
+    event.preventDefault();
+    setLoading(true);
+    try {
+      signupUser(formState.email, formState.password,
+        ROLE_NAMES[role],
+        formState.org, parseInt(formState.zipcode, 10), `${formState.firstName} ${formState.lastName}`,
+        formState.phone, formState.comments);
+    } catch (err) {
+      if (err) {
+        setErrorMsg(err);
+      }
+      console.log('Error in signupOnboarding');
+      console.log(err);
+    }
+    setFormState(INITIAL_FORM_STATE);
+    setLoading(false);
+    setCurrentStep(currentStep + 1);
+  }, [formState, role, currentStep]);
+
+  const onNext = useCallback(() => {
+    if (currentStep >= 3) {
```

Comment:
I just had it >= instead of == in case there was some lag or something and they managed to press the button twice. I wanted to make sure there was no undefined behavior